sample/spec/spec_helper.rb

```diff
@@ -20,4 +20,7 @@
   config.use_transactional_fixtures = true
 
   config.include FactoryGirl::Syntax::Methods
+
+  config.order = :random
+  Kernel.srand config.seed
```

Comment:
so the point is to be able to run specs again in the same order when necessary? 
again, can't you do it without that Kernel line? 
I might be wrong, but I don't have this line in my projects and I can do that.